examples/mnist_stages/config.yml

```diff
@@ -98,6 +98,9 @@ stages:
       optimizer:
         _target_: OptimizerCallback
         metric_key: *model_loss
+        grad_clip_fn: clip_grad_value_
```

Comment:
could you please also add a test for `catalyst/tests/teste_finetune` or something similar?